src/xtopcom/xtopcl/src/api_method.cpp

```diff
@@ -3890,7 +3890,7 @@ void ApiMethod::change_trans_mode(bool use_http) {
         trans_base::s_defaule_mode = TransMode::WS;
         // std::cout << "Using trans mode - WS: " << g_server_host_port  << std::endl;
     }
-#ifdef DEBUG
+#if  defined(DEBUG) || defined(RELEASEDEBINFO)
```

Comment:
the following code is designed for DEBUG only.
why enable it by adding RELEASEDEBINFO?  reldbg is not for debug dump, it's just making postmortem analysis a little bit easy.

Reply:
test need it on testcase conference